inc/actions.php

```diff
@@ -240,10 +240,11 @@ function uploadcare_settings() {
 /*
  * Add Uploadcare settings page to admin
  */
-add_action('admin_menu', 'uploadcare_settings_actions');
-function uploadcare_settings_actions() {
-    add_options_page('Uploadcare', 'Uploadcare', 'upload_files', 'uploadcare', 'uploadcare_settings');
-}
+add_action( 'admin_menu', 'uploadcare_add_admin_menu' );
```

Comment:
Please, follow code style from the rest of the file. Spaces around arguments not needed.